integrations/k8s-using-daemonset/k8s-with-rbac/falco-daemonset-configmap.yaml

```diff
@@ -25,7 +25,7 @@ spec:
 #          env:
 #          - name: SYSDIG_BPF_PROBE
 #            value: ""
-          args: [ "/usr/bin/falco", "-K", "/var/run/secrets/kubernetes.io/serviceaccount/token", "-k", "https://$(KUBERNETES_SERVICE_HOST)", "-pk"]
+          args: [ "/usr/bin/falco", "--cri", "/host/var/run/docker.sock", "-K", "/var/run/secrets/kubernetes.io/serviceaccount/token", "-k", "https://$(KUBERNETES_SERVICE_HOST)", "-pk"]
```

Comment:
I don't think we want to set this to the docker socket, but rather `/host/run/containerd/containerd.sock` or the crio socket path. 

 Also we need to include the appropriate host mounts to give the falco container access to `/run/containerd/containerd.sock` and the crio socket. A sample is below.

```
          volumeMounts:
            - mountPath: /host/var/run/docker.sock
              name: docker-socket
            - mountPath: /host/run/containerd/containerd.sock
              name: containerd-socket
            - mountPath: /host/dev
              name: dev-fs
...
        - name: docker-socket
          hostPath:
            path: /var/run/docker.sock
        - name: containerd-socket
          hostPath:
            path: /run/containerd/containerd.sock
        - name: dev-fs
          hostPath:
            path: /dev
```

Reply:
Yup you're right!  
Thanks